vision/getting-started/image_editing.ipynb

```diff
@@ -1,710 +1,1247 @@
 {
```

Comment:
<p>"You must set 1 to 5 classes" should be "You must set between 1 and 5 different classes" for extra clarity.</p><p> </p><p>"The full table of available classes is listed in the <code>Appendix</code> section at the end of this Colab."</p><p>The link should just be included here so a user doesn't have to scroll all the way down and come back here.</p><p> </p><p>FYI "Colab" as in "end of this <strong>Colab</strong>." should instead be "<strong>notebook</strong>", which is the industry-appropriate term.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/506/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
I will update this one in a future change